tests/test_run_length.py

```diff
@@ -37,15 +37,15 @@ def test_simple(self):
         )
         values[1:11] = 1
         da = xr.DataArray(values != 0, coords={"time": time}, dims="time")
-        lt = da.resample(time="M").apply(rl.longest_run_ufunc)
+        lt = da.resample(time="M").map(rl.longest_run_ufunc)
```

Comment:
What's the major difference? Is it the same method in the backend?

Reply:
I don't know! But there is a deprecation warning on `apply`, so why not.

Reply:
According to the Pandas documentation, `.map` is meant for general operations along the axis of interest but it looks like they have different behaviours depending on the Array/Series structure. Xarray probably follows this relatively closely. See: https://stackoverflow.com/questions/19798153/difference-between-map-applymap-and-apply-methods-in-pandas

Reply:
I just did some quick research and it looks like the name "map" was thought to be more general and intuitive... http://xarray.pydata.org/en/stable/whats-new.html#id52 5th point.